SocialCareCaseViewerApi/V1/Boundary/Requests/FinishCaseSubmissionRequest.cs

```diff
@@ -0,0 +1,21 @@
+using System.Text.Json.Serialization;
+using FluentValidation;
+
```

Comment:
you could add `#nullable enable` here.

https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare/

It will give us that benefit that unless we define a property as null it can not be set to null (or the compiler will complain)